DTO/src/engine2ui/simulation/genral/impl/objects/DTOEntity.java

```diff
@@ -0,0 +1,61 @@
+package engine2ui.simulation.genral.impl.objects;
+
+import engine2ui.simulation.genral.impl.properties.property.api.DTOProperty;
+
+public class DTOEntity {
+    private final String name;
+    private final int startingPopulation;
+    private final int endingPopulation;
+    private final DTOProperty[] properties;
```

Comment:
Most DTOs will now contain arrays instead of Lists since we don't add anything to the DTO after its created.